ci/stress_tests/application_cluster_template.yaml

```diff
@@ -90,10 +87,9 @@ file_mounts: {
 # List of shell commands to run to set up nodes.
 setup_commands:
     - echo 'export PATH="$HOME/anaconda3/envs/tensorflow_<<<PYTHON_VERSION>>>/bin:$PATH"' >> ~/.bashrc
-    - ray || wget https://s3-us-west-2.amazonaws.com/ray-wheels/releases/<<<RAY_VERSION>>>/<<<RAY_COMMIT>>>/ray-<<<RAY_VERSION>>>-<<<WHEEL_STR>>>-manylinux1_x86_64.whl
-    - rllib || pip install -U ray-<<<RAY_VERSION>>>-<<<WHEEL_STR>>>-manylinux1_x86_64.whl[rllib]
-    - pip install tensorflow-gpu==1.12.0
-    - echo "sudo halt" | at now + 60 minutes
+    - wget --quiet https://s3-us-west-2.amazonaws.com/ray-wheels/releases/<<<RAY_VERSION>>>/<<<RAY_COMMIT>>>/ray-<<<RAY_VERSION>>>-<<<WHEEL_STR>>>-manylinux1_x86_64.whl
+    - source activate tensorflow_p36 && pip install -U ray-<<<RAY_VERSION>>>-<<<WHEEL_STR>>>-manylinux1_x86_64.whl[rllib]
```

Comment:
How did this work before without the source activate? Should we just put the source activate in the bashrc instead of the export command above?

Reply:
This is switching to the DLAMI, which is the better AMI (tm).

Reply:
From the diff, it looks like it was already using the DLAMI.

Reply:
Ah, I see it's due to the export line above. That's bad practice, to inject lines, since it is implicitly changing the environment by side-effect.